.github/workflows/build.yml

```diff
@@ -8,36 +8,31 @@ on:
     release:
         types: [ created ]
     schedule:
-        -   cron: "0 1 * * 6" # Run at 1am every Saturday
+        - cron: "0 1 * * 6" # Run at 1am every Saturday
     workflow_dispatch: ~
 
 jobs:
     tests:
-        runs-on: ubuntu-18.04
+        runs-on: ubuntu-20.04
 
         name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: [ "8.0", "7.4" ]
-                symfony: [ "^4.4", "^5.2" ]
-                sylius: ["~1.9", "~1.10", "~1.11"]
-                node: [ ">=12.13.0" ]
-                mysql: [ "8.0" ]
+                php: [ "8.0" ]
+                symfony: ["^5.4", "^6.0"]
+                sylius: ["~1.11.0", "~1.12.0"]
+                node: ["^14.17.x"]
+                mysql: [ "5.7", "8.0" ]
 
                 exclude:
-                    - php: 8.0
-                      sylius: ~1.9
+                    - # Segmentation fault while clearing cache
```

Comment:
Is still current? As I remember, @rafalmal removed that conflict in another PRs and it passed.

Reply:
Removed in next commit